pkg/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -44,6 +44,12 @@ func New(profile *performancev1alpha1.PerformanceProfile) (*machineconfigv1.Kube
 		kubeletConfig.ReservedSystemCPUs = string(*profile.Spec.CPU.Reserved)
 	}
 
+	if profile.Spec.NUMA != nil {
+		if profile.Spec.NUMA.TopologyPolicy != nil {
+			kubeletConfig.TopologyManagerPolicy = string(*profile.Spec.NUMA.TopologyPolicy)
```

Comment:
I understand there are some specific values allowed for this, right? We should validate them in https://github.com/openshift-kni/performance-addon-operators/blob/master/pkg/controller/performanceprofile/components/profile/profile.go#L20

Reply:
Validations added

Reply:
I wonder if kubelet has an enum somewhere. Just to prevent us forgeting to adapt in case they add something new

Reply:
Golang doesn't have enums :man_shrugging: 
But there are constants available at least, see other comment.